junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/extension/ExtensionRegistry.java

```diff
@@ -60,6 +62,10 @@
 	public static ExtensionRegistry createRegistryWithDefaultExtensions() {
 		ExtensionRegistry extensionRegistry = new ExtensionRegistry(null);
 		DEFAULT_EXTENSIONS.forEach(extensionRegistry::registerDefaultExtension);
+		if (Boolean.getBoolean(EXTENSIONS_AUTODETECT_PROPERTY_NAME)) {
```

Comment:
Such properties must be looked up via `ConfigurationParameters`, not manually via `Boolean.getBoolean(...)`.